content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,249 @@
+
+## Prerequisites
+
+Before getting started, ensure you have the following prerequisites:
+
+1. An [AWS][7] account. Your AWS user needs the following IAM permissions to successfully run the CloudFormation template:
+
+    * cloudformation:CreateStack
+    * ec2:DescribeSecurityGroups
+    * ec2:DescribeSubnets
+    * ec2:DescribeVpcs
+    * iam:AttachRolePolicy
+    * iam:CreatePolicy
+    * iam:CreateRole
+    * iam:PassRole
+    * iam:PutRolePolicy
+    * iam:UpdateAssumeRolePolicy
+    * kms:Decrypt
+    * lambda:AddPermission
+    * lambda:CreateFunction
+    * lambda:GetCodeSigningConfig
+    * lambda:GetFunction
+    * lambda:GetFunctionCodeSigningConfig
+    * lambda:InvokeFunction
+    * lambda:PutFunctionConcurrency
+    * logs:CreateLogGroup
+    * logs:DescribeLogGroups
+    * logs:PutRetentionPolicy
+    * s3:CreateBucket
+    * s3:GetObject
+    * s3:GetObjectVersion
+    * secretsmanager:CreateSecret
+    * secretsmanager:GetSecretValue
+    * secretsmanager:PutSecretValue
+    * serverless:CreateCloudFormationTemplate
+
+
+## Setup
+
+
+2. From the AWS tile on the [Integrations page][8] in your Datadog account, select the Datadog products you wish to integrate with this AWS Account. This selects the correct default settings for integrating data for those products. These settings can be changed in the future if needed.
+{{< img src="getting_started/integrations/cloudformation-setup.png" alt="The Datadog AWS integration tile showing the options for establishing the integration. The Role Delegation tab is highlighted.">}}
+
+3. Select the AWS Region where the CloudFormation stack will be launched. This also sets where to create the Datadog Lambda Forwarder for sending AWS logs to Datadog (if you selected Log Management).
+
+    **Note**: CloudWatch metrics are collected from ALL AWS regions you are using regardless of the region you select.
+
+4. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+
+5. Click "Launch CloudFormation Template". This opens the AWS Console and loads the CloudFormation stack. All the parameters are filled in based on your selections in the prior Datadog form, so you do not need to edit those unless desired.
+**Note:** The `DatadogAppKey` parameter enables the CloudFormation stack to add and edit the Datadog configuration for this AWS account by using API calls to Datadog. The key is automatically generated and tied to your Datadog account.
+{{< img src="getting_started/integrations/params.png" alt="The AWS CloudFormation create-stack page showing the Stack name as datadog, IAMRoleName as DatadogIntegrationRole, ExternalId as an obfuscated value ending in be46, DdApiKey as an obfuscated value.">}}
+
+7. Check the required boxes from AWS and click `Create stack`:
```

Comment:
The template parameters have changed so this one is now more accurate. There are no more "advanced options"
![Screen Shot 2022-03-04 at 9 18 31 AM](https://user-images.githubusercontent.com/6912096/156779960-2a0baf17-d178-4701-966d-83d29eb26ab7.png)